Makefile.am

```diff
@@ -25,7 +25,6 @@ libfastjsoninclude_HEADERS = \
 
 libfastjson_la_LDFLAGS = -version-info 3:0:0 -no-undefined @JSON_BSYMBOLIC_LDFLAGS@ \
 	-export-symbols-regex '^fjson_.*'
-libfastjson_la_CPPFLAGS = -Werror
```

Comment:
What exactly is the problem with passing -Werror via CFLAGS? Do you have any references?

That said, a good middle ground could be something like [1].
During development you could build with
--enable-more-warnings=error.

The important part is, that hard-coding -Werror is bad and should be removed. 

[1] https://cgit.freedesktop.org/NetworkManager/NetworkManager/tree/m4/compiler_warnings.m4